dockerizing-notes.md

```diff
@@ -19,6 +20,7 @@
 
 ## TODO:
 
-* figure out what is needed for also developing `cg-style` (team usually uses `npm link`)
 * figure out why the shrinkwrap doesn't work -- looks like it has to do
```

Comment:
whoops, I had gotten this working over in #1036, but forgot to remove from these notes. Is it not working as part of `setup-npm-link.sh`?